eng/WpfArcadeSdk/tools/DrtsToRun.props

```diff
@@ -1,9 +1,32 @@
 <Project>
   <ItemGroup>
     <!--
-    List of Drts to run that are built out of dotnet/wpf. Note that this is incredibly
+    List of Drts (grouped by area) to run that are built out of dotnet/wpf. Note that this is incredibly
```

Comment:
Think about this mechanism being expanded to FeatureTests.  It could get pretty unwieldy.

Maybe we won't have a similar style of things going on there, but we should probably be sure that what mechanisms we use for DRTs are very similar to FeatureTests.  Maybe these should be automatically generated by a target parsing the QV files (like DrtList.xml and DiscoveryInfo.xml).

Maybe the best course of action would be using QV itself to generate the list that gets batched into Helix because then we could apply the discovery mechanism with specific variables that can trim/alter the test list.

Probably not a *now* thing, but might be something to think about going forward.

Reply:
what do we do for feature tests, is there a FeatureTestList.xml like DRTs? My plan is to generate this file based on that list, which should be fairly straightforward to do

Reply:
Unfortunately FeatureTests aren't that straightforward.  In the same directory with DrtList, is DiscoveryInfo,xml.  This doesn't list everything though, you need to use QV to get the full test discovery I believe, which is how I generated that excel file we had with the test lists.

I had to hack QV for that, but you can probably add an option to spit out the discovery list and we can run it as a one off.

@miguep None of this sounds completely incorrect does it?  : - )